docs/Changelog.md

```diff
@@ -0,0 +1,882 @@
+* Apply modern C# features/syntaxes [#1183](https://github.com/microsoft/testfx/pull/1183)
+* Rename MSTest.CoreAdapter into MSTest.TestAdapter [#1181](https://github.com/microsoft/testfx/pull/1181)
+* Move test projects to SDK style [#1179](https://github.com/microsoft/testfx/pull/1179)
+  
+### Fixed
+  
+* Fix broken tests and refactor test API [#1352](https://github.com/microsoft/testfx/pull/1352)
+* Fix issue causing null ref when test class has no namespace [#1283](https://github.com/microsoft/testfx/pull/1283)
+* Fix localization path
+* Fix .gitignore
+* Fix included package [#1280](https://github.com/microsoft/testfx/pull/1280)
+* Fix common.lib.ps1 to download latest patched SDKs [#1279](https://github.com/microsoft/testfx/pull/1279)
+* Fix included package [#1280](https://github.com/microsoft/testfx/pull/1280)
+* Fix common.lib.ps1 to download latest patched SDKs [#1279](https://github.com/microsoft/testfx/pull/1279)
+* Fix broken unit tests
+* Fix testasset of ComponentTests
+* Fix some typos in test names
+* Fix some typos
+* Fix NuGet packages content + support netcoreapp3.1 [#1228](https://github.com/microsoft/testfx/pull/1228)
+* Fix broken tests on main [#1265](https://github.com/microsoft/testfx/pull/1265)
+* Fixed the case ignoring in AreEqual() with culture parameter [#1216](https://github.com/microsoft/testfx/pull/1216)
+* Fix behavior for netcore TFMs [#1230](https://github.com/microsoft/testfx/pull/1230)
+* Fixed package restore.
+  
+### Removed
+  
+* Remove skipping check for x64 [#1350](https://github.com/microsoft/testfx/pull/1350)
+* Remove class/assembly initialization messages from logs. [#1339](https://github.com/microsoft/testfx/pull/1339)
+* Remove unexpected dll in target [#1308](https://github.com/microsoft/testfx/pull/1308)
+* Remove PlatformServices.Shared.Unit.Tests [#1270](https://github.com/microsoft/testfx/pull/1270)
+* Remove unused build step [#1266](https://github.com/microsoft/testfx/pull/1266)
+* Remove PlatformServices.Portable [#1258](https://github.com/microsoft/testfx/pull/1258)
+* Remove un-needed files
+* Remove ProjectGuid and simplify ProjectReference calls [#1185](https://github.com/microsoft/testfx/pull/1185)
+* Remove test adapter nuget uap dependencies [#1195](https://github.com/microsoft/testfx/pull/1195)
+* Remove broken UWP unit test [#1295](https://github.com/microsoft/testfx/pull/1295)
+
+### Artifacts
+
+* MSTest: [3.0.0-preview-20221110-04](https://www.nuget.org/packages/MSTest/3.0.0-preview-20221110-04)
+* MSTest.TestFramework: [3.0.0-preview-20221110-04](https://www.nuget.org/packages/MSTest.TestFramework/3.0.0-preview-20221110-04)
+* MSTest.TestAdapter: [3.0.0-preview-20221110-04](https://www.nuget.org/packages/MSTest.TestAdapter/3.0.0-preview-20221110-04)
+
+## [2.3.0-preview-20220810-02] 2022-08-10
+
+* [x] [Fixed issues with SDK style projects.](https://github.com/microsoft/testfx/pull/1171)
```

Comment:
For here and below, you can remove:

1. the `[x]` part
2. the blank line

And you can also add the `Added/Changed...` sections reading from the title of the commit.